dnf-behave-tests/features/plugins-core/repomanage.feature

```diff
@@ -0,0 +1,202 @@
+Feature: Tests for repomanage command
+
+
+Background:
+Given I enable plugin "repomanage"
+
+
+Scenario: basic functionality of repomanage --new
+Given I copy repository "dnf-ci-thirdparty-updates" for modification
+ When I execute dnf with args "repomanage --new {context.dnf.repos[dnf-ci-thirdparty-updates].path}"
+ Then the exit code is 0
+  And stdout is
+      """
+      {context.dnf.tempdir}/repos/dnf-ci-thirdparty-updates/src/CQRlib-extension-1.6-2.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-thirdparty-updates/src/SuperRipper-1.3-1.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-thirdparty-updates/x86_64/CQRlib-extension-1.6-2.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-thirdparty-updates/x86_64/SuperRipper-1.3-1.x86_64.rpm
+      """
+
+
+Scenario: basic functionality of repomanage --new works without repodata
+Given I copy repository "dnf-ci-thirdparty-updates" for modification
+  And I delete directory "/{context.dnf.repos[dnf-ci-thirdparty-updates].path}/repodata"
```

Comment:
It actually has to be there, this way the path start with `//` which means it [doesn't get prepended with installroot](https://github.com/rpm-software-management/ci-dnf-stack/blob/master/dnf-behave-tests/common/lib/file.py#L24), that is what we need since the `{context.dnf.repos[dnf-ci-thirdparty-updates].path}` already contains the installroot path.

Reply:
I really dislike this btw. Thinking about it maybe we could prepend installroot only if the path doesn't start with a format placeholder. Because I think it almost always should if it's not an installroot path.

Reply:
I agree, that could work, paths not starting with format placeholder really shouldn't be pointing to the host and thus can be prepended.

But this pr is complex enough, so I don't want to do this kind of change here.

Reply:
You are right, my bad.